fv3core/stencils/dyn_core.py

```diff
@@ -199,12 +213,6 @@ def compute(state, comm):
             if global_config.get_do_halo_exchange():
                 reqs["delp_quantity"].wait()
                 reqs["pt_quantity"].wait()
-            beta_d = 0
-        else:
-            beta_d = spec.namelist.beta
```

Comment:
beta_d will matter if we have a namelist with beta > 0... but I guess we don't need to have those in here right now